src/HomePage.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import Button from "./Button";
+import Menu from "./Menu";
+import {connect} from "react-redux";
+
+
+class HomePage extends React.Component {
+    constructor(p) {
+        super(p);
+        console.log(this.props.CurrPage)
+    }
+
+    OnPageChange(pageId) {
+        window.location = pageId
+    }
+
+    getRootClassName() {
+        return this.props.CurrPage;
+    }
+
+    getContentClassName() {
+        const totnString = this.props.CurrPage;
+        return (totnString || "").concat("Content");
+    }
+
+    render() {
+        return (
+                <div className={this.getRootClassName()}>
+                    <div className={this.getContentClassName()}>
+                        <h1>{this.props.language === "ENG" ? "Do you want a dog?" : "Хочеш собачку?" }</h1>
+                        <div className="btns">
+                            <Button
+                                    Caption={this.props.language === "ENG" ? "Why a dog?" : "Чому собачка?"}
```

Comment:
so one big thing in React: 
component/class names should start with upper case, everything else with lower case. So your props should be in default camelCase: `caption`, `pageName`, `onChange`, etc.